pkg/cmd/cli/cmd/set/env.go

```diff
@@ -273,14 +382,45 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				c.Env = updateEnv(c.Env, env, remove)
 
 				if list {
+					resolveErrors := map[string][]string{}
+					store := newResourceStore()
+
 					fmt.Fprintf(out, "# %s %s, container %s\n", info.Mapping.Resource, info.Name, c.Name)
```

Comment:
i didn't notice that before, but that would create a problem for --list w/o --resolve in that the output isn't shell script parseable, even though there would be no error.